### PR TITLE
Update dev  readme

### LINKED ISSUE #14: Add steps to creating Rundeck project on dev Readme
During local development, the steps to creating a [project](https://rundeck.org/docs/administration/projects/create-new-project.html) on the Rundeck server running as a container locally, should be written in the dev [readme](https://github.com/kosyfrances/rundeck_zabbix/blob/master/dev/README.md).

To change the Resource Model Document Format to yaml instead of the default xml, exec into the running rundeck container and update this file `/etc/rundeck/project.properties`, specifically these fields:
```
resources.source.1.config.file=/var/rundeck/projects/${project.name}/etc/resources.yml
resources.source.1.config.format=resourceyml
```

To create a project, visit `http://localhost:4440/resources/createProject`. Enter the project name and click `create`, then `save`, leaving the defaults for now.

### PR DESCRIPTION
**Pull request by kosyfrances**
Fixes #14 

Add steps to creating Rundeck project

### CODE REVIEW

File: dev/README.md [hunk 1]
```diff
@@ -44,6 +44,16 @@ Verify that the value of `Template App HTTP Service: HTTP service is down on {HO
 
 ![Photo of Zabbix Host Trigger Page](assets/zabbix-trigger-page.png)
 
+### Create Rundeck project
+To create a [new project](https://docs.rundeck.com/docs/administration/projects/create-new-project.html) on Rundeck;
```
Comment (osule) [inline dev/README.md]: Perhaps this sentence can be made clearer.
Does a new project have to be created in Rundeck before going on to do the next steps?

File: dev/README.md [hunk 2]
```diff
@@ -44,6 +44,16 @@ Verify that the value of `Template App HTTP Service: HTTP service is down on {HO
...
+* Change the Resource Model Document Format to yaml instead of the default xml (we only support yaml in our codebase for now). Exec into the running rundeck container and update this file `/etc/rundeck/project.properties`, specifically these fields:
+```
+resources.source.1.config.file=/var/rundeck/projects/${project.name}/etc/resources.yml
+resources.source.1.config.format=resourceyml
```
Comment (osule) [inline dev/README.md]: Can confirm the format in this snippet?
`resourceyml` looks unlikely.

File: dev/README.md (hunk 1 above)
Comment (kosyfrances) [reply dev/README.md]: It is just a link for the user if they want more information. This is mostly for local development

File: dev/README.md (hunk 2 above)
Comment (kosyfrances) [reply dev/README.md]: It is actually supposed to be `resourceyaml` https://docs.rundeck.com/2.9.1/administration/project-setup.html#resource-model-document-formats

**Pull request merged**